util.js

```diff
@@ -224,79 +225,38 @@ function fetch(url, options) {
-                    }
-                    if (/\/respec\//i.test(resource.url.path)) {
-                        fetch(resource.url.href)
-                            .then(response => response.text())
-                            .then(data => callback(null, data))
-                            .catch(err => callback(err));
-                    }
-                    else if ((resource.url.pathname === '/webcomponents/assets/scripts/autolink.js') ||
-                        (resource.url.href.startsWith(baseUrl) &&
-                            !(/annotate_spec/i.test(resource.url.pathname)) &&
-                            !(/link-fixup/i.test(resource.url.pathname)) &&
-                            !(/bug-assist/i.test(resource.url.pathname)) &&
-                            !(/dfn/i.test(resource.url.pathname)) &&
-                            !(/section-links/i.test(resource.url.pathname)) &&
-                            !(/^\/webidl\//i.test(resource.url.pathname)))) {
-                        fetch(resource.url.href)
-                            .then(response => response.text())
-                            .then(data => callback(null, data))
-                            .catch(err => callback(err));
-                    }
-                    else {
-                        callback(null, '');
-                    }
-                },
-                done: (err, window) => {
-                    if (err) {
-                        return reject(err);
-                    }
-                    const doc = window.document;
-                    const links = doc.querySelectorAll('body > .head a[href]');
-                    for (let i = 0 ; i < links.length; i++) {
-                        let link = links[i];
-                        let text = (link.textContent || '').toLowerCase();
-                        if (text.includes('single page') ||
-                            text.includes('single file') ||
-                            text.includes('one-page')) {
-                            let singlePage = URL.resolve(doc.baseURI, link.getAttribute('href'));
-                            if (singlePage === url) {
-                                // We're already looking at the single page version
-                                resolve(window);
-                            }
-                            else {
-                                loadSpecification(singlePage)
-                                    .then(window => resolve(window))
-                                    .catch(err => reject(err));
-                            }
-                            return;
+            const {window} = new JSDOM(html, {url: response.url});
+            const doc = window.document;
+            // ReSpec doc
+            if (doc.head.querySelector("script[src*='respec']")) {
+                // this does another network fetch :(
+                return respecWriter(url, '/dev/null', {}, 20000).then(function(html) {
```

Comment:
`/dev/null` does not mean anything on Windows and makes the code crash. Using an empty string `''` seems to do what you want.